src/io/flutter/editor/WidgetIndentsHighlightingPass.java

```diff
@@ -710,31 +770,39 @@ private void buildWidgetDescriptors(
     if (outlineNode == null) return;
 
     final String kind = outlineNode.getKind();
-    final boolean widgetConstructor = "NEW_INSTANCE".equals(kind);
+    final boolean widgetConstructor = "NEW_INSTANCE".equals(kind) || (parent != null && ("VARIABLE".equals(kind)));
 
     final List<FlutterOutline> children = outlineNode.getChildren();
-    if (children == null || children.isEmpty()) return;
+    //    if (children == null || children.isEmpty()) return;
```

Comment:
should this line be removed?

Reply:
that optimization can be added back. history is there is a separate block of work to render previews of property values inline split off from this cl.